policykit/slackintegration/views.py

```diff
@@ -137,6 +151,22 @@ def is_policykit_action(integration, test_a, test_b, api_name):
 
     return False
 
+#creates slackUser object along with the readable name and user_id
```

Comment:
you should remove this

Reply:
this was the start of my solution to #6 which I don't think I got working yet, but I will send a new request without it